Formula/moarvm.rb

```diff
@@ -24,8 +25,8 @@ class Moarvm < Formula
   conflicts_with "rakudo-star", because: "rakudo-star currently ships with moarvm included"
 
   resource "nqp" do
```

Comment:
If it turns out we want the versions to always match, and we have a formula for nqp, does this resource block still need to exist?

Reply:
I'm not at all familiar with these formulae but I would think that this could be replaced with `depends_on "nqp"`.

Reply:
Oh I think it's because nqp depends on moarvm.